Makefile

```diff
@@ -0,0 +1,2 @@
+generate_panel_instructions:
```

Comment:
It looks like you're checking the generated files in - wondering if they should be gitignored

Reply:
this is just a tool to bootstrap the markdown - the markdown are the actual files we want persisted in source